app/javascripts/main/networking.ts

```diff
@@ -15,56 +23,48 @@ const pipeline = promisify(pipelineFn);
 export async function downloadFile(url: string, filePath: string) {
   await ensureDirectoryExists(path.dirname(filePath));
   const response = await get(url);
-  await pipeline(response, fs.createWriteStream(filePath));
+  await pipeline(
+    /**
+     * IncomingMessage doesn't implement *every* property of ReadableStream
+     * but still all the ones that pipeline needs
+     * @see https://www.electronjs.org/docs/api/incoming-message
+     */
+    response as any,
```

Comment:
Would `response as Partial<ReadableStream>` work?

Reply:
It's possible, but it doesn't offer any more guarantees than casting as `any`. I believe it's a more explicit way to tell that we're bypassing the type checker this way.

Reply:
I think the advantage of some type checking over none is that if something changes here in the future, a flag would be thrown.